blink.ino

```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
+  [-delay(1000);-]{+delay(3000);+}               // wait for a second
```

Comment:
```suggestion
  [-delay(1000);-]{+delay(3000);+}               // wait for a second
//i believe that it would be better if the light was off for 4 seconds and on for just 3